### PR TITLE
Sdh guidetitlefixes

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **Updated the "Sign Your Users in to" titles ** 
- **No**

### CODE REVIEW

File: packages/@okta/vuepress-site/guides/protect-your-api/next-steps/index.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
 ---
 title: Next Steps
 ---
-Now that you can protect your API, you may need additional information on other topics to help with other tasks.
```
Comment (jakubvul) [inline packages/@okta/vuepress-site/guides/protect-your-api/next-steps/index.md]: I don't think exclamation points fit our tone in the docs.

Comment (jakubvul) [reply packages/@okta/vuepress-site/guides/protect-your-api/next-steps/index.md]: Actually, the whole phrase "Nice job!", exclamation point or not, seems out of place.

File: packages/@okta/vuepress-site/guides/sign-into-mobile-app/before-you-begin/iOS/create-app.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+The [iOS quickstart](android link here) teaches you the basics of building an Android application.
```
Comment (jakubvul) [inline packages/@okta/vuepress-site/guides/sign-into-mobile-app/before-you-begin/iOS/create-app.md]: Well that's confusing...

File: packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-in-page/android/signin.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-First initialize the Okta AppAuth SDK in the `Activity#onCreate` method of the Activity that you are using to log users into your app. In this example, we call it LoginActivity:
+First initialize the Okta AppAuth SDK in the `Activity#onCreate` method of the Activity that you are using to log users in to your app. In this example, we call it LoginActivity:
```
Comment (jakubvul) [inline packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-in-page/android/signin.md]: ~log~ sign

File: packages/@okta/vuepress-site/guides/sign-into-mobile-app/before-you-begin/iOS/create-app.md (hunk 2 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/guides/sign-into-mobile-app/before-you-begin/iOS/create-app.md]: Ha, yeah - this is just a placeholder for the iOS quickstart...I'll update the text.

File: packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-in-page/android/signin.md (hunk 3 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-in-page/android/signin.md]: done

File: packages/@okta/vuepress-site/guides/protect-your-api/next-steps/index.md (hunk 1 above)
Comment (nbarbettini) [reply packages/@okta/vuepress-site/guides/protect-your-api/next-steps/index.md]: This was my attempt at a friendly tone, but I'm not tied to it. If you don't like it, yank it.

**Pull request merged**